src/Illuminate/Redis/Connections/PredisClusterConnection.php

```diff
@@ -2,7 +2,19 @@
 
 namespace Illuminate\Redis\Connections;
 
+use Predis\Command\ServerFlushDatabase;
+
 class PredisClusterConnection extends PredisConnection
 {
-    //
+    /**
+     * Flush the selected Redis database.
+     *
+     * @return void
+     */
+    public function flushdb()
+    {
+        foreach ($this->getConnection()->getIterator() as $node) {
```

Comment:
As of Redis v1.1.0, we can iterate `Predis\Client` directly. @bernardwiesner What Predis version are you running?

Let's update this to:

```php
foreach ($this->client as $node) {
    $node->executeCommand(new ServerFlushDatabase);
}
```